examples/free_convection_notes.md

```diff
@@ -0,0 +1,18 @@
+# Convection into a linearly stratified fluid
+
+We are performing 'large eddy simulations' (actually, direct numerical 
+simulations with artificially elevated viscosity and diffusivity)
+of convection into a stratified fluid. 
+Our parameters regime corresponds roughly to conditions that
+occur in the North Atlantic ocean in winter.
+
+## Parameters:
+
+* ν    10^(-2)       (m^2 / s),   viscocity
+* κ    10^(-2)       (m^2 / s),   thermal diffusivity
```

Comment:
Oceananigans (and the MITgcm) generally differentiates between horizontal and vertical diffusivities and viscosities so might be good to mention that you're setting both to the same value.

Reply:
Perhaps.

But actually I think it is the anisotropic case that deserves special mention. The isotropic case should be default. Ocean water has isotropic viscosity and diffusivities.

Ultimately I think we should classify an anisotropic viscosity as a 'closure' (or 'subgrid model' --- whatever we end up calling these things that are extensions to Navier-Stokes). We can implement an `AnisotropicViscosityClosure` when we develop our 'subgrid model' abstraction. I believe this implementation is appropriate for an LES code. I would also like to emphasize to user X that using anisotropic viscosity really amounts to using a kind of subgrid model (with all the errors and caveats that come with approximate models) for turbulence.

Reply:
Ah that makes sense to me. Would be nice if it was clearly used as a sub-grid-scale model. But this will have to wait for another refactor.